src/fragments/Loading/spinners/CircleSpinner.jsx

```diff
@@ -28,7 +28,7 @@ const CircleSpinner = ({
     return (
         <div style={style ? style : {}} className={spinnerClass}>
             {debugTitle}
-            <div className="dash-sk-circle">
+            <div className="dash-spinner dash-sk-circle">
```

Comment:
Extra class on the spinners, easier to target general spinner through css (and to automate tests)